src/messaging/message_bus_bull.ts

```diff
@@ -129,6 +129,7 @@ export class MessageBusBull extends MessageBusBase {
 	}
 
 	protected async onInit(): Promise<void> {
+		await super.onInit();
```

Comment:
Instead
```typescript
await super.onInit();
....LOCAL INIT CODE....
``` 
we have to write something like following
```typescript
await super.onInit();

try {
    ....LOCAL INIT CODE....
}
catch(e) {
    await super.onDispose();
    throw e;
}
```
to prevent leaks if `....LOCAL INIT CODE....` failure

Let's speak about by voice...

Reply:
Fix